### DISCUSSION TITLE
Open Source HLS Backend

### DISCUSSION THREAD

**Discussion by JochiSt** [fastmachinelearning/hls4ml#721] [Ideas]
I'm wondering myself, whether there is already any implementation of HLS4ML, which is targeting a fully open source toolchain (e.g. Yosys or similar).

This might be an interesting feature for the Open Source ASIC chains as well.

If there is no target backend, which open source HLS do you recommend?

**Comment by sachinkum0009** [comment]
Sounds pretty cool, I am also interested in this.
